tests/system/__init__.py

```diff
@@ -12,5 +12,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import time
+
 KIND = "SomeKind"
 OTHER_NAMESPACE = "other-namespace"
+
+
+def eventually(predicate, timeout=30, interval=1):
```

Comment:
Do you think this could be used somewhere else than in system tests? Not sure if it's worth to have as part of the API if we are only going to use it in tests.

Reply:
I hadn't thought of that. In the interest of keeping the surface we have to support as small as possible, let's leave this here. If it becomes apparent, later, that users might like to use this as a general utility, we can always move it.